pages/browse/index.vue

```diff
@@ -17,36 +16,8 @@ const { data: browseSections, pending } = useBrowse();
         ></li>
       </ul>
     </template>
-    <ul v-else>
-      <template v-for="section in browseSections" :key="section.id || 0">
-        <PageHeading v-if="section.type === 'section_header'" :level="3">
-          <div class="flex items-center justify-between">
-            <div>
-              <a v-if="typeof section.link === 'string'" :href="section.link">
-                {{ section.title }}
-              </a>
-              <span v-else>{{ section.title }}</span>
-            </div>
-            <a v-if="section.link" :href="section.link">
-              <ButtonStyled intent="secondary" size="small">
-                <span class="whitespace-nowrap">
-                  {{ t("home.list.see-all") }}
-                </span>
-              </ButtonStyled>
-            </a>
-          </div>
-        </PageHeading>
-        <AlbumItem v-else-if="section.type === 'album'" :album="section" />
-        <PlaylistItem
-          v-else-if="section.type === 'playlist'"
-          :playlist="section"
-        />
-        <PodcastItem
-          v-else-if="section.type === 'podcast'"
-          :podcast="section"
-        />
-        <p v-else style="color: red">This is not implemented yet</p>
-      </template>
-    </ul>
+    <template v-else>
+      <DocumentModels v-if="models !== null" :models="models"></DocumentModels>
```

Comment:
Why not use the component `DocumentList` here?

Reply:
I don't understand the question